src/main/java/ch/njol/skript/expressions/ExprItemWithPotionEffect.java

```diff
@@ -0,0 +1,177 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.ChatColor;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.PotionMeta;
+import org.bukkit.inventory.meta.SuspiciousStewMeta;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.Aliases;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+
+@Name("Item with Potion Effects")
+@Description("Get an item with potion effects. Supports potions, splash potions, lingering potions, tipped arrows, and suspicious stew.")
+@Examples({"give player potion with blindness and nausea of tier 3 with duration 10 minutes",
+	"set {_i} to splash potion with health", "set player's tool to a tipped arrow with slowness of tier 3 with duration 5 minutes"})
+@RequiredPlugins("1.14.4+ for Suspicious Stew")
+@Since("INSERT VERSION")
+public class ExprItemWithPotionEffect extends SimpleExpression<ItemType> {
+	
+	static {
+		if (Skript.classExists("org.bukkit.inventory.meta.SuspiciousStewMeta")) {
+			Skript.registerExpression(ExprItemWithPotionEffect.class, ItemType.class, ExpressionType.PROPERTY,
```

Comment:
This is a combined expression, not a property one.

Reply:
Ill make sure to fix this